Update torusUserGuide.md

danelsuga opened the pull request:
I was wondering if this should remain xDai wallet balance on line 28 since it’s referring to the currency not the network. Thanks.
Lots of capital changes here you might want to check out too! :)

laurenluz reviewed (changes requested):
Hey @danelsuga! Great capitalization fixes & edits. However, when ` is used instead of "... it creates the button formatting. I think mitch did this on purpose and I don't think we should change it. Can you take those out? 

Maybe you can add @divine-comedian to review this after you make those adjustments

![image](https://user-images.githubusercontent.com/75490971/152841489-5da333a3-f88e-40e2-8ad6-f00975f3ff8a.png)

danelsuga commented:
Oh crud. Of course. Sorry. I was actually JUST thinking about this. I only made that mistake on this one page, I remember. Will fix!

danelsuga commented:
OK, I updated the patch by correcting the quotation marks.

I noted however that a number of them do not present as buttons. Is that meant to be the case? (As an aside, if so, it’s more standard to use double quotes but no big deal to leave it —just wanted to point out in case they are also supposed to be buttons.)

The very first one for example does not present as a button: ‘wallet top-up'. 

This leads me to suspect that’s why I changed them all. Sorry! I subsequently had the original page on a separate screen when I edited anything else (on every document; not just this one) in order to follow along correctly.

I also noticed that a double quote was used once on this page (« Connect Wallet »), so I have now made that a single quote for consistency.

@divine-comedian could you please take a look? Thanks!

divine-comedian commented:
> OK, I updated the patch by correcting the quotation marks.
> 
> I noted however that a number of them do not present as buttons. Is that meant to be the case? (As an aside, if so, it’s more standard to use double quotes but no big deal to leave it —just wanted to point out in case they are also supposed to be buttons.)
> 
> The very first one for example does not present as a button: ‘wallet top-up'.
> 
> This leads me to suspect that’s why I changed them all. Sorry! I subsequently had the original page on a separate screen when I edited anything else (on every document; not just this one) in order to follow along correctly.
> 
> I also noticed that a double quote was used once on this page (« Connect Wallet »), so I have now made that a single quote for consistency.
> 
> @divine-comedian could you please take a look? Thanks!

I think you used the wrong quotation marks - the formatting is enabling by wrapping them in ` not '
